contracts/generate_artifacts.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
is it maybe more intuitive to put this script inside the /solidity folder ?

Reply:
@FabijanC exactly, even if they are not used directly for compiling (at least for now), you can now find the associated actions [in the script](https://github.com/glihm/starknet-devnet-rs/blob/7dd5d6e0fe8f14aca40ebd562cc3c5d5e16ce873/contracts/generate_artifacts.sh).
Also, running this script is now **mandatory** before building Devnet. The readme was adjusted accordingly.

Reply:
From my understanding, for general L1-L2 support, we only need the `MockStarknetMessaging` contract artifact. In devnet-py, those artifacts necessary for startup (i.e. necessary for users) were kept as a part of the git repo. However, those needed for testing (and potential example showcasing) were expected to be generated using a script.

I'd keep it the same way here. Currently [the documentation that a new user sees upon arriving to the repo](https://github.com/0xSpaceShard/starknet-devnet-rs#run) is wrong (incomplete): it is not sufficient to just come to the repo and do `cargo run`, you are also expected to run pre-build scripts.

Reply:
That's a good point. The artifact is not re-introduced in 5208964828148554b1d5ef1102f60cc025605893.